### DISCUSSION TITLE
Question about the #operator for the table (#Configs.mapping)

### DISCUSSION THREAD

**Discussion by smithx** [linrongbin16/gitlinker.nvim#44] [Q&A]
[Operator](https://github.com/linrongbin16/gitlinker.nvim/blob/master/lua/gitlinker.lua#L110) returns 0 on all my systems (linux and windows), neovim 0.9.1 & 0.10.0. So, no one key mapping is installing, although the table contains 2 elements (by default)

PR #45

**Comment by linrongbin16** [comment]
Hi, please let me first check this case.

What is your `mapping` that can re–produce this issue?

**Comment by linrongbin16** [comment]
let's track it in #48
